alcs-frontend/src/app/features/notice-of-intent/decision/conditions/condition/condition.component.ts

```diff
@@ -0,0 +1,104 @@
+import { AfterViewInit, Component, Input, OnInit } from '@angular/core';
+import moment from 'moment';
+import { UpdateApplicationDecisionConditionDto } from '../../../../../services/application/decision/application-decision-v2/application-decision-v2.dto';
+import { NoticeOfIntentDecisionConditionService } from '../../../../../services/notice-of-intent/decision-v2/notice-of-intent-decision-condition/notice-of-intent-decision-condition.service';
+import { UpdateNoticeOfIntentDecisionConditionDto } from '../../../../../services/notice-of-intent/decision/notice-of-intent-decision.dto';
+import {
+  DECISION_CONDITION_COMPLETE_LABEL,
+  DECISION_CONDITION_INCOMPLETE_LABEL,
+  DECISION_CONDITION_SUPERSEDED_LABEL,
+} from '../../../../../shared/application-type-pill/application-type-pill.constants';
+import { CONDITION_STATUS, ConditionComponentLabels, DecisionConditionWithStatus } from '../conditions.component';
+
+type Condition = DecisionConditionWithStatus & {
+  componentLabelsStr?: string;
+  componentLabels?: ConditionComponentLabels[];
+};
+
+@Component({
+  selector: 'app-condition',
+  templateUrl: './condition.component.html',
+  styleUrls: ['./condition.component.scss'],
+})
+export class ConditionComponent implements OnInit, AfterViewInit {
+  @Input() condition!: Condition;
+  @Input() isDraftDecision!: boolean;
+  @Input() fileNumber!: string;
+
+  incompleteLabel = DECISION_CONDITION_INCOMPLETE_LABEL;
+  completeLabel = DECISION_CONDITION_COMPLETE_LABEL;
+  supersededLabel = DECISION_CONDITION_SUPERSEDED_LABEL;
+
+  CONDITION_STATUS = CONDITION_STATUS;
+
+  isReadMoreClicked = false;
+  isReadMoreVisible = false;
+  conditionStatus: string = '';
+  isRequireSurveyPlan = false;
+
+  constructor(private conditionService: NoticeOfIntentDecisionConditionService) {}
+
+  ngOnInit() {
+    this.updateStatus();
+    if (this.condition) {
+      this.condition = {
+        ...this.condition,
+        componentLabelsStr: this.condition.conditionComponentsLabels?.flatMap((e) => e.label).join(';\n'),
+      };
+
+      this.isRequireSurveyPlan = this.condition.type?.code === 'RSPL';
```

Comment:
since there are no lots anymore I think this one is redundant

Reply:
Removed